### DISCUSSION TITLE
Увеличить размер доступной памяти

### DISCUSSION THREAD

**Discussion by berlinsky-nk** [1c-syntax/bsl-language-server#3352] [Q&A] [answered]
Помогите разобраться, как _правильно_ сделать сабж? В доке и настройках не нашёл, только тут, а оно подтянется потом в новую версию?

%USERPROFILE%\AppData\Roaming\Code\User\globalStorage\1c-syntax.language-1c-bsl\bsl-language-server\v0.23.1\bsl-language-server\app\bsl-language-server.cfg

![image](https://github.com/user-attachments/assets/8bb3a3b6-6f67-4b9f-9645-545c1247f366)

На проектах 20k+ файлов плагин крашится при стандартных 3g.
![image](https://github.com/user-attachments/assets/79815b07-e648-4146-9498-d6e885e8c652)

**Comment by nixel2007** [comment, marked as answer]
Добрый день. Да, если вы используете exe-версию, то надо править этот конфигурационный файл. К сожалению, в новую версию не перенесется, надо будет заново повышать.

У вас чистая ерп или с какими-то нашлепками?

**Reply by berlinsky-nk** [reply]
УХ, нашлёпок файлов на 100.

**Reply by nixel2007** [reply]
Разжирела уха ~~при собянине~~... Надо будет поглядеть, раньше она в 2 гига помещалась. А сейчас не умещается в 3

@theshadowco

**Reply by berlinsky-nk** [reply]
ЦФ - 1G без поставки, исходники вроде 6 :) 100k файлов

**Reply by nixel2007** [reply]
Мне кажется, года три назад уха была 16к  bsl файлов. А теперь 21к. Но это не точно :)
